graylog2-web-interface/src/components/event-notifications/event-notification-types/EmailNotificationForm.jsx

```diff
@@ -50,37 +50,6 @@ Last messages accounting for this alert:
 \${end}
 `;
 
-// TODO: Default html body template should come from the server
```

Comment:
I've meant to keep this one, actually.
Having a nice default template is great. I just didn't want to force it on the users for existing notifications.
But it should be filled in when creating a new notification.

Reply:
Oh, sorry I misunderstood it